stylelint.config.js

```diff
@@ -27,9 +30,260 @@ module.exports = {
       }
     },
     "order/order": [
-      "custom-properties",
-      "declarations"
+      [
+				"dollar-variables",
+				"custom-properties",
+				"at-rules",
+				"declarations",
+				{
+					type: "at-rule",
+					name: "media",
+        },
+        "rules",
+			]
     ],
+    "order/properties-order": [
+			[
+				"content",
+				"position",
+				"top",
+				"right",
+				"bottom",
+				"left",
+				"z-index",
+				"display",
+				"vertical-align",
+				"flex",
+				"flex-grow",
+				"flex-shrink",
+				"flex-basis",
+				"flex-direction",
+				"flex-flow",
+				"flex-wrap",
+				"grid",
+				"grid-area",
+				"grid-template",
+				"grid-template-areas",
+				"grid-template-rows",
+				"grid-template-columns",
+				"grid-row",
+				"grid-row-start",
+				"grid-row-end",
+				"grid-column",
+				"grid-column-start",
+				"grid-column-end",
+				"grid-auto-rows",
+				"grid-auto-columns",
+				"grid-auto-flow",
+				"grid-gap",
+				"grid-row-gap",
+				"grid-column-gap",
+				"gap",
+				"row-gap",
+				"column-gap",
+				"align-content",
+				"align-items",
+				"align-self",
+				"justify-content",
+				"justify-items",
+				"justify-self",
+				"order",
+				"float",
+				"clear",
+				"object-fit",
+				"overflow",
+				"overflow-x",
+				"overflow-y",
+				"overflow-scrolling",
+				"clip",
+
+				//
+				"box-sizing",
+				"width",
+				"min-width",
+				"max-width",
+				"height",
+				"min-height",
+				"max-height",
+				"margin",
+				"margin-top",
+				"margin-right",
+				"margin-bottom",
+				"margin-left",
+				"padding",
+				"padding-top",
+				"padding-right",
+				"padding-bottom",
+				"padding-left",
+				"border",
```

Comment:
Border is before padding

Reply:
Border and all it's properties were moved up.